<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatchSettingsPanel</class>
 <widget class="QDialog" name="PatchSettingsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Patches configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Skipping</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBox_skiphudmemos">
          <property name="text">
           <string>Skip item collected messages</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_skipfrigate">
          <property name="text">
           <string>Skip Pirate Frigate</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_skipcrater">
          <property name="text">
           <string>Skip Impact Crater</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Difficulty</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
         <widget class="QRadioButton" name="radioStackProtection">
          <property name="text">
           <string>Stack with suits</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Heat damage protection:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Damage reduction:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="radioStaggerProtection">
          <property name="text">
           <string>Always use most powerful</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radioHeatDamageAny">
          <property name="text">
           <string>Any except Power Suit</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="radioHeatDamageVaria">
          <property name="text">
           <string>Varia Suit only</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox_bomblockpick">
          <property name="text">
           <string>Power bomb lock-picking</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Misc</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_fixflaaghra">
        <property name="text">
         <string>Fix Flaaghra music files</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Metroid Prime Trilogy disk:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineFlaaghra">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="browseButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_keepattractmode">
        <property name="text">
         <string>Keep Attract mode</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PatchSettingsPanel</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PatchSettingsPanel</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
